src/planner.cc

```diff
@@ -277,6 +278,9 @@ class Planner::Impl {
   // segments and update only the last deceleration ramp.
   int num_segments_ready_ = 0;
 
+  // Number of maximum planning steps allow to enqueue.
+  int lookahead_size_ = PLANNING_BUFFER_CAPACITY;
```

Comment:
I'd make this `size_t` so that you don't have to use the awkward unsigned cast in Line 470.
This value is vetted by the `SetLookahead()` function already to make sure it never is negative.